tools/platforms/WebAssemblyPlatform.hx

```diff
@@ -481,42 +462,20 @@ class WebAssemblyPlatform extends PlatformTarget
 			}
 		}
 
-		for (asset in project.assets)
-		{
-			var path = Path.combine(targetDirectory + "/obj/assets", asset.targetPath);
-
-			if (asset.type != AssetType.TEMPLATE)
-			{
-				// if (asset.type != AssetType.FONT) {
-
-				System.mkdir(Path.directory(path));
-				AssetHelper.copyAssetIfNewer(asset, path);
-
-				// }
-			}
-		}
-
 		ProjectHelper.recursiveSmartCopyTemplate(project, "webassembly/template", destination, context);
 		ProjectHelper.recursiveSmartCopyTemplate(project, "haxe", targetDirectory + "/haxe", context);
 		ProjectHelper.recursiveSmartCopyTemplate(project, "webassembly/hxml", targetDirectory + "/haxe", context);
 		// ProjectHelper.recursiveSmartCopyTemplate(project, "webassembly/cpp", targetDirectory + "/obj", context);
 		ProjectHelper.recursiveSmartCopyTemplate(project, "cpp/static", targetDirectory + "/obj", context);
 
-		for (asset in project.assets)
-		{
-			var path = Path.combine(destination, asset.targetPath);
-
-			if (asset.type == AssetType.TEMPLATE)
-			{
-				System.mkdir(Path.directory(path));
-				AssetHelper.copyAsset(asset, path, context);
-			}
-		}
+		copyProjectAssets(destination, targetDirectory + "/obj/assets");
 	}
 
 	@ignore public override function install():Void {}
 
 	@ignore public override function trace():Void {}
 
 	@ignore public override function uninstall():Void {}
+
+	@ignore public override function watch():Void {}
```

Comment:
`watch()` relies on `getDisplayHXML()`, which is missing from `AIRPlatform` and `TizenPlatform`, so that explains why we don't have it there.

However, `WebAssemblyPlatform` has `getDisplayHXML()` and doesn't have `watch()`. I don't know why, but I kept that behavior anyway. It seems like it ought to work, if we just enable it.